src/GitLinkTask/GitLink.cs

```diff
@@ -10,21 +10,18 @@ namespace GitLinkTask
     using System;
     using System.IO;
     using System.Linq;
-    using GitLink;
-    using GitLink.Pdb;
-    using GitLink.Providers;
+    using global::GitLink;
+    using global::GitLink.Pdb;
+    using global::GitLink.Providers;
     using Catel.Logging;
     using Microsoft.Build.Framework;
     using Microsoft.Build.Utilities;
 
-    public class LinkProject : Task
+    public class GitLink : Task
```

Comment:
Is it wise to call the task the same? Is this the one used in msbuild? Or can it be added via an attribute? If so, then I prefer GitLinkTask where we define the xml name via attributes.